### PR TITLE
Add F-Droid repository announcement

### PR DESCRIPTION
**Pull request by TheAssassin**
Thanks to @gkeegan and @B0pol for helping with the article.

We will have to add some more in-depth instructions in the FAQ section or something comparable later. @gkeegan made a few very good screenshots which we can use then.

### CODE REVIEW

**Comment by Stypox** [comment]
Maybe we should also show how to manually add a repository to Aurora Droid, some people use it because it works and looks better than the original fdroid client, but it has a slightly different interface for adding repositories.

**Comment by wb9688** [comment]
@Stypox: But then we also have to write for G Droid and all the others. Imho we should only write instructions for F-Droid and generally users of alternative frontends should know how to use those.

**Comment by B0pol** [comment]
Plus it's really straightforward to add a repo in aurora droid. Drawer -> repo -> qrcode that is clearly shown or paste the url. On top of that, when you install it tells you how to add a repo.

**Comment by TheAssassin** [comment]
I also don't think we need to explain it for all these alternative clients in this blog post. We can add those information in the FAQ or whatever additional resource we're going to provide.

File: _posts/2020-05-30-f-droid-repo.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+The easiest way to add the repository is to scan the QR code below or [click this link](fdroidrepo://archive.newpipe.net/fdroid/repo/?fingerprint=E2402C78F9B97C6C89E97DB914A2751FDA1D02FE2039CC0897A462BDB57E7501) if you're viewing this on your device already. F-Droid should pop up on both, prompting you to add the repository. For the paranoid (*which is not a bad idea, especially in this context!*), you can verify the fingerprint F-Droid shows you with the one in the "copy-pastable" URL below.
+
+<center>
+    <img src="/img/fdroid-repo-qrcode.svg" alt="NewPipe F-Droid repository QR code" style="width: 250px">
+</center>
+
+Alternatively, in F-Droid, open the *Settings*, and choose *Repositories*. Then press the *plus* button, and paste the link if needed.
+
+Once you added the repository, update your package lists. If you had NewPipe installed from the official F-Droid repository before, you will have to uninstall this one before F-Droid will display the packages from our repository.
```
Comment (TobiGr) [inline _posts/2020-05-30-f-droid-repo.md:28]:
```suggestion
Once you added the repository, update your package lists. If you had NewPipe installed from the official F-Droid repository before, you will have to uninstall this one before F-Droid will display the packages from our repository. Please make sure to [backup and restore the ap data]({{ site.base_url }}/FAQ/tutorials/import-export-data/#export-subscriptions) to not lose your subscriptions, history and settings.

```

File: _posts/2020-05-30-f-droid-repo.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+The URL you need to copy is below. It includes the key's fingerprint for your convenience.
```
Comment (TobiGr) [inline _posts/2020-05-30-f-droid-repo.md:30]: Why is this placed down here? The instructions to add the repository two paragraphs above.

File: _posts/2020-05-30-f-droid-repo.md (hunk 1 above)
Comment (B0pol) [reply _posts/2020-05-30-f-droid-repo.md:28]: not to lose your subscriptions* (tell me if I'm wrong)

**Comment by wb9688** [comment]
Like @TobiGr said on IRC, it would probably be a good idea to link https://f-droid.org/en/tutorials/add-repo/.

>Thanks to @gkeegan and @B0pol for helping with the article.

Why not add them as co-authors of the commit then?

**Comment by gkeegan** [comment]
I did not write any of this iirc. I just provided some screenshots that could be used in the more detailed explanation (as described on line 38) if one is made.

File: _posts/2020-05-30-f-droid-repo.md (hunk 1 above)
Comment (Poolitzer) [reply _posts/2020-05-30-f-droid-repo.md:28]: in order to avoid loosing your XXX

**Comment by TobiGr** [comment]
There were problems with the QR code as @gkeegan found out. I was able to fix this by using the URL to the repo instead of the `fdroidrepo/` intend

File: _posts/2020-05-30-f-droid-repo.md (hunk 2 above)
Comment (TheAssassin) [reply _posts/2020-05-30-f-droid-repo.md:30]: The recommended way is to use the QR code. This is just a fallback.

Comment (wb9688) [reply _posts/2020-05-30-f-droid-repo.md:31]: @TheAssassin: Above (line 20) is a clickable link, so imho there is no need for a text link.

Comment (TheAssassin) [reply _posts/2020-05-30-f-droid-repo.md:31]: Less tech-savvy users might not understand how to copy a link. So having something in plain text they can copy is nice. Also, it allows for introspecting the link (on the browser, I can move the mouse over a link and wait a few milliseconds, but on a smartphone this doesn't work...).

Comment (wb9688) [reply _posts/2020-05-30-f-droid-repo.md:31]: >Less tech-savvy users might not understand how to copy a link.

You don't need to copy it, just click it and it'll open the F-Droid app.

>on the browser, I can move the mouse over a link and wait a few milliseconds, but on a smartphone this doesn't work...

On a smartphone you could long press it.

Comment (gkeegan) [reply _posts/2020-05-30-f-droid-repo.md:31]: The line 20 link cannot be pasted into the repo box without editing out the `fdroidrepo://` part, which is why having the plain link without it is nice.

**Pull request merged**